### PR TITLE
UX revisions to drill down header

### PR DESCRIPTION
**Pull request by ngraham76**
# Description

* Removed the kebab menu icon button next to the output dropdown as we no longer need the Save function here - it's now been added to operator's output header as a `Save for re-use` button
* Added a 1px bottom border (surface-border-alt) to the operator drilldown headers - defined in our Figma designs, was missing

image

### CODE REVIEW

File: packages/client/hmi-client/src/components/drilldown/tera-drilldown-header.vue [hunk 1]
```diff
@@ -1,5 +1,5 @@
 <template>
-	<header>
+	<header class="border-b" :style="{ borderColor: 'var(--surface-border-alt)' }">
```
Comment (YohannParis) [inline packages/client/hmi-client/src/components/drilldown/tera-drilldown-header.vue]: You can just change line 46 to have a different border-color

File: packages/client/hmi-client/src/components/drilldown/tera-drilldown.vue [hunk 2]
```diff
@@ -59,10 +59,6 @@
 							:output="selectedOutputId"
 							@update:selection="(e) => emit('update:selection', e)"
 						/>
-						<section v-if="!isEmpty(menuItems)" class="mx-2">
-							<Button icon="pi pi-ellipsis-v" rounded text @click.stop="toggleEllipsisMenu" />
-							<Menu ref="ellipsisMenu" :model="menuItems" popup />
```
Comment (YohannParis) [inline packages/client/hmi-client/src/components/drilldown/tera-drilldown.vue:64]: You can remove the prop menuItems now that is not used anymore.

**Review by YohannParis — APPROVED** [review]

File: packages/client/hmi-client/src/components/drilldown/tera-drilldown-header.vue (hunk 1 above)
Comment (ngraham76) [reply packages/client/hmi-client/src/components/drilldown/tera-drilldown-header.vue]: Oh. Right. :) I'll do that now.

File: packages/client/hmi-client/src/components/drilldown/tera-drilldown.vue (hunk 2 above)
Comment (ngraham76) [reply packages/client/hmi-client/src/components/drilldown/tera-drilldown.vue:64]: image

**Review by mwdchang — CHANGES_REQUESTED** [review]
We need to clean up the callers so they no longer generate and send in "menu-items" unnecessarily.

**Comment by mwdchang** [comment]
TA3 operators likely needs a bit of design tweaks for removing these menu-item paradigms, as they are saving different types of asset or assets. See follow up in:

https://github.com/DARPA-ASKEM/terarium/issues/4713

**Review by mwdchang — APPROVED** [review]

**Pull request merged**